samples/unit-tests/indicator-sma/recalculations/demo.js

```diff
@@ -306,15 +306,15 @@ QUnit.test('Order of series and indicators, #15892.', function (assert) {
             enabled: false
         },
         series: [{
-            id: 'main',
-            data: [13, 14, 15, 13, 14, 15, 13, 14, 15]
-        },
-        {
             type: 'sma',
+            id: 'sma',
             linkedTo: 'main',
             params: {
                 period: 4
             }
+        }, {
+            id: 'main',
+            data: [13, 14, 15, 13, 14, 15, 13, 14, 15]
```

Comment:
Q: Why change the order of the series?

Reply:
It's actually not related to the PR - I think I should have mentioned it in commit message. There was [a test](https://github.com/highcharts/highcharts/blob/d85eeae99daa78343ca26317a431dcd47291d68f/samples/unit-tests/indicator-sma/recalculations/demo.js#L324) that was supposed to check if the indicator declared before main series is initialized correctly. The thing is, old chart config had main series declared before indicator and I have changed the order so that the test makes sense now.

Reply:
Nice catch! Thanks